src/localization.ts

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 import {SourceMapConsumer} from 'source-map';
 import * as fs from 'fs';
 
-interface Location {
+export interface Location {
```

Comment:
should there be a difference between a web location (URL) and a disk location ? I know they are similar, but separating them will make the code easier to follow.